backend/dev/app.ts

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import { router } from './router.js';
+
+const app = express();
+let port:number = process.env.PORT === undefined ? 4200 : +process.env.PORT;
```

Comment:
it might be easier

process.env.PORT || 4200

why let? are we planning to change the variable?